iothub_client/src/iothub_client_ll_uploadtoblob_uhttp.c

```diff
@@ -254,7 +254,7 @@ static HTTP_CLIENT_HANDLE create_http_client(IOTHUB_CLIENT_LL_UPLOADTOBLOB_HANDL
         uhttp_client_destroy(result);
         result = NULL;
     }
-    else if (upload_data->curl_verbosity_level != UPOADTOBLOB_CURL_VERBOSITY_UNSET && uhttp_client_set_trace(result, true, true) != HTTP_CLIENT_OK)
+    else if (upload_data->curl_verbosity_level == UPOADTOBLOB_CURL_VERBOSITY_ON && uhttp_client_set_trace(result, true, true) != HTTP_CLIENT_OK)
```

Comment:
Will this break our customers applications?

Reply:
There are no existing apps - this implementation is for uhttp only and exists only in this branch.